doc/python/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2016, EPFL/Blue Brain Project
+#                     Juan Hernando Vieites <juan.hernando@epfl.ch>
+#
+# This file is part of Brion <https://github.com/BlueBrain/Brion>
+#
+
+# Reusing the FindSphinx script from Pydoxine
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/Pydoxine/CMake)
+
+common_find_package(Sphinx)
```

Comment:
I can only do that if I move the CMAKE_MODULE_PATH append also to the top because FindSphinx.cmake comes from Pydoxine.

Then, the question is which version should be required? For generating the docstrings Sphinx 1.3 is needed, but for generating the Python documentation a smaller version can also work. If I remove default_docstrings.cpp I can move this to the top level and request 1.3.
If I keep default_docstrings.cpp I can still move this up, but I haven't tested if the Sphinx detection done by Pydoxine (which happens when PythonDocstrings.cmake is included) still works as expected.